### ISSUE TITLE
IncludeXmlComments provides parameter-description twice (with FromQuery complex parameters)

### ISSUE THREAD

**Issue by runehalfdan** [domaindrivendev/Swashbuckle.AspNetCore#1612]
After upgrading from Swashbuckle.AspNetCore 5.0.0 to 5.3.1 (I think it happens with 5.2.x as well) many of my parameters have two (identical) descriptions. One for parameter itself, once for schema

Replacing  
```
options.IncludeXmlComments(xmlPath);
```
with
```
xmlDoc = new XPathDocument(xmlPath);
options.ParameterFilter<XmlCommentsParameterFilter>(xmlDoc);
options.RequestBodyFilter<XmlCommentsRequestBodyFilter>(xmlDoc);
options.OperationFilter<XmlCommentsOperationFilter>(xmlDoc);
```

helps as a workaround. But I don't like it

Details "description": "test name" is repeated 
```
"parameters": [
          {
            "name": "$name",
            "in": "query",
            "description": "test name",
            "schema": {
              "type": "string",
              "description": "test name",
              "nullable": true
            }
          },
          {
            "name": "$dummy",
            "in": "query",
            "description": "Test dummy",
            "schema": {
              "type": "array",
              "items": {
                "type": "string"
              },
              "description": "Test dummy",
              "nullable": true
            }
          }
        ],
```
Entrypoint:
```
[HttpGet("cache/update")]
public async Task<ActionResult<string>> TestStuffAsync([FromQuery] StuffDto queryParam)
```
StuffDto:
```
public class StuffDto
    {
        /// <summary>
        /// test name
        /// </summary>
        [FromQuery(Name = "$name")]
        public string Name { get; set; }

        /// <summary>
        /// Test dummy
        /// </summary>
        [FromQuery(Name = "$dummy")]
        public IEnumerable<string> Stuff { get; set; }
    }
```
StuffDto isn't reference anywhere also in the code

**Comment by redoz** [comment]
Can confirm, the workaround posted has some other drawbacks that makes it less suitable for my scenario, but this hack seems to have done the trick:
```csharp
public class SwaggerDescriptionFilter : IParameterFilter
{
    public void Apply(OpenApiParameter parameter, ParameterFilterContext context)
    {
        if (parameter.Description == parameter.Schema.Description)
            parameter.Schema.Description = null;
    }
}
```

**Comment by domaindrivendev** [comment]
Looks like this was fixed by c1fb18c2c2f864285a7b811a147d8a9ed8c17993, and will be available with the upcoming `6.0.0` release. In the meantime you can pull down the latest preview build from [myget.org](https://myget.org/feed/domaindrivendev/package/nuget/Swashbuckle.AspNetCore) to try it out.
